environments/production/public.yml

```diff
@@ -171,7 +171,7 @@ localsettings:
   ES_MULTIPLEX_TO_VERSION: '6'
   # Index Multiplexer Settings  
   ES_APPS_INDEX_MULTIPLEXED: False
-  ES_CASE_SEARCH_INDEX_MULTIPLEXED: True
+  ES_CASE_SEARCH_INDEX_MULTIPLEXED: False # Index swapped
```

Comment:
Should `ES_CASE_SEARCH_INDEX_SWAPPED: True` be reset to `False` now that this is done?

Reply:
Nope. We will need keep Swapped as `True` until we change primary and secondary index names in consts file in es app.